### ISSUE TITLE
Add tooltips to hangout status icons explaining what "solo study" vs. "silent study" vs. "collaborative study" means

### ISSUE THREAD

**Issue by lpatmo** [codebuddiesdotorg/cb-v2-scratch#18]
See: http://getbootstrap.com/javascript/#tooltips

**Comment by wuworkshop** [comment]
This pure CSS one might be useful - https://github.com/kazzkiq/balloon.css

**Comment by vishnubhagwan** [comment]
#232

**Comment by lpatmo** [comment]
@vishnubhagwan Sorry, should've described this issue better -- 

![screen shot 2016-09-29 at 1 10 39 am](https://cloud.githubusercontent.com/assets/4512699/18945950/9c16bfac-85e1-11e6-9344-c078488c08ca.jpg)

- [ ] When you hover over the red "silent" hangout icon, a popover should appear. Its content should say: ```Create a silent hangout to motivate yourself to get through a tutorial with others who are learning from the same tutorial, or create one for a fun silent coworking experience.```

- [ ] When you hover over the yellow "teaching" hangout icon, the popover should say: ```Create a teaching hangout to practice teaching a concept or to walk participants through a codebase. This is your chance to share what you know!```

- [ ] And finally, if you hover over the green "collaborative" hangout icon, the popover should say: ```Create a collaboration hangout to pair program on a coding exercise, talk through a chapter of a tutorial, or work on an open-sourced project together.```

An example of elsewhere we've used popovers is in the about.html file.

If you or anyone else would like to work on this issue, please leave a comment on this thread!

**Comment by BratCat** [comment]
I think I can handle this one if it's not already taken.

**Comment by lpatmo** [comment]
@BratCat Hi Marcia! Please hold for a minute... I think I saw another issue that looked very similar to this one, that had already received a pull request. Let me check that one first.

**Comment by BratCat** [comment]
@lpatmo will do.

**Comment by lpatmo** [comment]
@BratCat You are good to go! :) Turns out the other PR I was afraid overlapped only introduced the `title` attribute to the HTML tags. This issue is still needed, so this issue is yours if you're still interested in working on it. Thanks!

**Comment by BratCat** [comment]
@lpatmo ok thanks so much I'll go ahead an start working on it. Thanks!

**Comment by BratCat** [comment]
@lpatmo Hi! I had to create a new css file for the tooltips in update_status, it's update_status.scss...what file do I need to update for it show the styles? Right now I have a link in the html file. 

```
<template name="updateStatus">
  <link rel="stylesheet" href="_update_status.scss">
    <div class="status-input">
```

**Comment by lpatmo** [comment]
Hi Marcia! You need to import the file in style.scss. The <link rel> etc. is unnecessary because we have a meteor package that automatically looks for scss files in style.scss and compiles them. Hope that helps!

P.S. Am typing from phone so sorry for any inconsistencies.

**Comment by BratCat** [comment]
Ok, thank you!

**Comment by distalx** [comment]
resolved in #244.
